model/src/main/kotlin/utils/LicenseResolver.kt

```diff
@@ -78,7 +79,10 @@ internal class LicenseResolver(
             }
 
             val curatedLicenseFindings = curationMatcher.applyAll(rawLicenseFindings, curations)
-            val matchedFindings = findingsMatcher.match(curatedLicenseFindings, copyrightFindings).toSortedSet()
+            val decomposedFindings = curatedLicenseFindings.flatMap { finding ->
+                SpdxExpression.parse(finding.license).decompose().map { finding.copy(license = it.toString()) }
```

Comment:
Using `decompose()` here means that we can have license findings containing the `WITH` operator. AFAIK no scanner so far creates such license findings. Would these license findings be handled correctly in other parts of ORT? E.g. when looking for license texts in the notice reporter, or when passing licenses to rules in the evaluator?
If not, it's probably better to use `licenses()` here and ignore exceptions, and add support for those later.

Reply:
> Using decompose() here means that we can have license findings containing the WITH operator. AFAIK no scanner so far creates such license findings. 

Even though the scanner do not provide compound expression or one containing the WITH operator the idea is still to handle it as such, because then the handling of `raw detected` and `curated detected` licenses is equal. My primary goal are the curated license findings, not the findings from the scanner.

> Would these license findings be handled correctly in other parts of ORT? 

From the callers components I manually verified the evaluator / rules to be working, found no effect on the notices as exceptions are ignored anyway, the static HTML report looked good while the results are illustrated in below test case. Evaluated model and WebApp are not (yet) affected as the `EvaluatedModelMapper` has code redundant with the changed one (which is changed in the next commit).